lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1497,6 +1505,70 @@ def test_teamset_type(self, requesting_user, expected_status):
             self.assertEqual(team['name'], 'foo')
 
 
+@ddt.ddt
+class TestTeamAssignmentsView(TeamAPITestCase):
+    """ Tests for the TeamAssignmentsView """
+
+    @classmethod
+    def setUpClass(cls):
+        """ Create an openassessment block for testing """
+        super().setUpClass()
+
+        course = cls.test_course_1
+        teamset_id = cls.solar_team.topic_id
+
+        section = ItemFactory.create(
+            parent=course,
+            category='chapter',
+            display_name='Test Section'
+        )
+        open_assessment = ItemFactory.create(
```

Comment:
maybe toss another section in here without an openassessment so we know for sure the filtering works?

Reply:
or maybe an openassessment for another teamset

Reply:
what field is the uniqueness constraint on?

Reply:
For whatever reason it tries to make a duplicate student when I tweak the test setup at all...

Reply:
Figured it out.